relay-quotas/src/rate_limit.rs

```diff
@@ -11,26 +11,32 @@ use smallvec::SmallVec;
 use crate::quota::{DataCategories, ItemScoping, Quota, QuotaScope, ReasonCode, Scoping};
 use crate::REJECT_ALL_SECS;
 
-/// A monotonic expiration marker for `RateLimit`s.
+/// A monotonic expiration marker for rate limits.
 ///
-/// `RetryAfter` marks an instant at which a rate limit expires, which is indicated by `expired`. It
-/// can convert into the remaining time until expiration.
+/// [`RetryAfter`] represents a point in time when a rate limit expires. It allows checking
+/// whether the rate limit is still active or has expired, and calculating the remaining time
+/// until expiration.
+///
+/// This type is designed to work with monotonic time, ensuring consistent behavior even if
+/// the system clock changes.
```

Comment:
Why did you add this, are you sure this is even correct? What happens if the system suspends?

Reply:
Yeah, there are some cases which `Instant` doesn't guarantee motonicity. I will remove this.